packages/wallets/src/routes/Router.tsx

```diff
@@ -58,13 +72,43 @@ const Router: React.FC = () => {
     const { data: walletAccounts, isLoading } = useWalletAccountsList();
 
     if ((!walletAccounts || !walletAccounts.length) && !isLoading)
-        return <Route component={WalletNoWalletFoundState} path={walletsPrefix} />;
+        return (
+            <Route
+                path={walletsPrefix}
+                render={() => (
+                    <React.Suspense fallback={null}>
+                        <LazyWalletsNotFoundState />
+                    </React.Suspense>
+                )}
+            />
+        );
 
     return (
         <Switch>
-            <Route component={CompareAccountsRoute} path={`${walletsPrefix}/compare-accounts`} />
-            <Route component={CashierModalRoute} path={`${walletsPrefix}/cashier`} />
-            <Route component={WalletsListingRoute} path={walletsPrefix} />
+            <Route
+                path={`${walletsPrefix}/compare-accounts`}
+                render={() => (
+                    <React.Suspense fallback={null}>
```

Comment:
I'm curios to see the actual impact - e.g. see the sizes & order of chunks loaded before this change and after this change (e..g to see one big chunk being loaded before, and then how the chunks / order changed with this change). 
(also just to verify that its actually removed from main bundle now - I had cases when I made something lazy loaded, but one minor import somewhere still caused the original code to be included in main bundle)

Reply:
Sure, I ran Lighthouse for `/wallets/cashier/deposit` page and only the cashier chunks (`cashier-modal-route`) are loaded. Previously all of the chunks for CFD/accounts/cashier are included resulting in total size of 57MB, now its 42MB:
Before:
<img width="1728" alt="Screenshot 2024-04-02 at 3 35 53 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/225d97b7-9e08-432b-9802-f8dea44a08bd">
After:
<img width="1728" alt="Screenshot 2024-04-02 at 3 35 57 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/ccc0e009-253f-47b2-883c-c0ee1ae41a92">

Reply:
For wallets listing page `/wallets` (57MB -> 48MB):
Before:
<img width="1728" alt="Screenshot 2024-04-02 at 3 39 50 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/7bcc7986-6985-4b67-a9e5-c5592a178fa2">

After:
<img width="1728" alt="Screenshot 2024-04-02 at 3 39 14 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/fe3f5723-df6e-42eb-8cdf-554ebe647582">

For `/wallets/compare-accounts` (57MB -> 42MB):
Before:
<img width="1724" alt="Screenshot 2024-04-02 at 3 38 25 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/1253bff7-1933-4760-bdec-cddebc10293f">
After:
<img width="1728" alt="Screenshot 2024-04-02 at 3 38 28 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/1eaa3d4d-d624-4566-bb35-ce34cb19d221">